misc/conda/README.rst

```diff
@@ -19,11 +19,11 @@ To install miniconda in ``/opt/conda`` do
    ./install_conda.sh -y /opt/conda
 
 
-To create a conda environment called ``py39`` with default Python version (3.9) and without GPU support
+To create a conda environment called ``py310`` with default Python version (3.10) and without GPU support
 
 ::
 
-   ./make_conda_env.sh -y -e py39
+   ./make_conda_env.sh -y -e py310
```

Comment:
Maybe this env should be called `scico` to make it more consistent with `docs/source/contributing.rst`

Reply:
Done.